src/explore-education-statistics-frontend/src/components/CookieBanner.module.scss

```diff
@@ -0,0 +1,24 @@
+.container {
+  background: #d5e8f3;
+  padding: 10px 0;
+  width: 100%;
+
+  p {
+    font-family: 'nta', Arial, sans-serif;
```

Comment:
Have these styles been lifted from somewhere else? Shouldn't need to be re-defining `p` with things like `font-family` as they are already globally styled.